kong/clustering/compat/init.lua

```diff
@@ -348,8 +348,40 @@ function _M.update_compatible_payload(payload, dp_version, log_suffix)
         end
       end
     end
+
+    local config_upstream = config_table["upstreams"]
```

Comment:
should we check `dp_version_num` like others?

Reply:
I have checked this https://github.com/Kong/kong/pull/10192/files#diff-d83069a1914775b9ca1488dbad907f5296c3af09328f3d0daf3d345407f90638R337